### ISSUE TITLE
Not able to install

### ISSUE THREAD

**Issue by toshendra** [MetaMask/eth-block-tracker#22]
Hey,

It seems there is some issue in installation. It has been installed locally as well as globally (to ensure it works in any case). Still saying it is not found. 

```
My-Shell$ npm install eth-block-tracker
/Users/XXXXX/Desktop/webapp
└── eth-block-tracker@1.1.2 

```
```
My-Shell$ node main.js 
module.js:471
    throw err;
    ^

Error: Cannot find module 'eth-block-tracker'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/Toshendra/Desktop/webapp/main.js:3:22)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**The content of the main.js is:**
```
const HttpProvider = require('ethjs-provider-http')
const BlockTracker = require('eth-block-tracker')

const provider = new HttpProvider('https://mainnet.infura.io/<my unique code>')
const blockTracker = new BlockTracker({ provider })
blockTracker.on('block', console.log)
blockTracker.start()
```

What might have gone wrong??

Even Runkit is showing not found.

https://npm.runkit.com/eth-block-tracker?t=1497419620361

**Comment by Schaeff** [comment]
Seems like `npm run compile` is not being called.

**Comment by kumavis** [comment]
hello/
i ran into this yesterday but didnt have time to fix - on it now

**Comment by kumavis** [comment]
fixed in eth-block-tracker@1.1.3

**Comment by kumavis** [comment]
removed `.npmignore` which caused npm to default to `.gitignore`'s list
